cmd/meroxa/root/apps/deploy.go

```diff
@@ -977,7 +977,7 @@ func (d *Deploy) Execute(ctx context.Context) error {
 	d.specVersion = d.flags.Spec
 
 	// Make the latest version default for Ruby and Go
-	if d.specVersion == "" && (d.lang == ir.Ruby || d.lang == ir.GoLang) {
+	if d.specVersion == "" && (d.lang == ir.Ruby || d.lang == ir.GoLang || d.lang == ir.JavaScript) {
```

Comment:
This may be easier to read?

```suggestion
       supportedLangs =  d.lang == ir.Ruby || d.lang == ir.GoLang || d.lang == ir.JavaScript
	if d.specVersion == "" && supportedLangs {
```